src/new-client/src/Web/MainRoute.tsx

```diff
@@ -36,19 +41,38 @@ const MainWrapper = styled.div`
   overflow: hidden;
 `
 
-export const MainRoute: React.FC = () => (
-  <Wrapper>
-    <DisconnectionOverlay />
-    <AppLayoutRoot data-qa="app-layout__root">
-      <Header />
-      <MainWrapper>
-        <Resizer defaultHeight={30}>
-          <LiveRates />
-          <Trades />
-        </Resizer>
-        <Analytics />
-      </MainWrapper>
-      <Footer />
-    </AppLayoutRoot>
-  </Wrapper>
-)
+export const MainRoute: React.FC = () => {
+  const tearOutEntry = useTearOutSectionEntry()
+  const [tornOutSectionState, setTornOutSectionState] = useState({
```

Comment:
Same here, good practice to supply a type

Reply:
Done